index.html

```diff
@@ -244,7 +244,7 @@ <h3>Description Computation</h3>
             for the HTML elements which meet this condition.
           </td>
           <td>
-            Either a <a href="#mapping_additional_nd_te">text equivalent computation</a> of the element, or as a flat string of the attribute's value.
+            Either a <a href="#mapping_additional_nd_name">name computation</a> of the element, or as a flat string of the attribute's value.
```

Comment:
> 4. I think the original note (or some variant of it) is still useful to convey the intent. Something like:
> 
> ```
> <div class="note">A node cannot refer to its name or description with <pref>aria-describedby</pref> for the purpose of <a href="#mapping_additional_nd_te">description computation</a>. This differs from <pref>aria-labelledby</pref>, with which a node <em>can</em> refer to itself and concatenate its own <pref>aria-label</pref> value into its <a href="#mapping_additional_nd_name">name computation</a> (see the example in [need new named link that was previously referred to generically as "step 2D"]).</div>
> ```
I understand that this would be generally undesirable as it would mostly lead to duplicate announcements but how about the following terrible use case.
```
<button id="a" aria-describedby="b a c">Submit</button>
<span id="b">press</span>
<span id="c">to finish the process</span>
```
Note that I know this is something no one should ever do (i18n and they fact that it just seems wrong) but is there any need to forbid referencing oneself with aria-describedby? I can't think of a good use for it now but that doesn't mean that one doesn't exist.

Reply:
It's an informative note, so not forbidding anything.